### ISSUE TITLE
[ChoiceGroup][Accessibilty] Difference b/w focused and non foucsed element is not distinguishable

### ISSUE THREAD

**Issue by aadhars** [OfficeDev/office-ui-fabric-react#3621]
### Bug Report

- __Package version(s)__: Latest
- __Browser and OS versions__: Edge

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: (Blocking, High, Normal, Low) : Normal

Products/sites affected: (if applicable): VSTS 

#### Describe the issue:

Difference b/w focused and non focused choice group item is not easily distinguishable.  See Image below
![image](https://user-images.githubusercontent.com/29277148/34253141-6430c4c4-e66c-11e7-94dc-9f050202aeab.png)
Although the  border width is increased by 1px but it is very hard to distinguish.

#### Actual behavior:

Not able to distinguish focused and non focused items

#### Expected behavior:

Focused and Non focused choice group item should be easily distinguishable. We make make the border dotted in high contrast. That should work 

### If applicable, please provide a codepen repro:

https://developer.microsoft.com/en-us/fabric#/components/choicegroup

**Comment by betrue-final-final** [comment]
Another one @lynamemi

**Comment by lynamemi** [comment]
@betrue-final-final  What would be a desirable design for this?

**Comment by betrue-final-final** [comment]
There needs to be a gap between the item and the keyboard focus. Here's a closeup of the design file.
<img width="829" alt="screen shot 2018-02-02 at 2 36 39 pm" src="https://user-images.githubusercontent.com/20363037/35758092-fea54a80-0826-11e8-89f1-2e38de06d93a.png">

and selection plus focus for extra clarity
<img width="395" alt="screen shot 2018-02-02 at 2 36 06 pm" src="https://user-images.githubusercontent.com/20363037/35758102-0b4c7006-0827-11e8-8101-7bb6c4966c47.png">

**Comment by lynamemi** [comment]
Should tack the hover state onto this effort: https://github.com/OfficeDev/office-ui-fabric-react/issues/4533

**Comment by michaelangotti** [comment]
@lynamemi Make sure to close this manually once PR #4752 is merged. Thanks!
